osu.Game.Rulesets.Osu/Difficulty/Skills/OsuStrainSkill.cs

```diff
@@ -57,15 +55,7 @@ public override double DifficultyValue()
                 strains[i] *= Interpolation.Lerp(ReducedStrainBaseline, 1.0, scale);
             }
 
-            // Difficulty is the weighted sum of the highest strains from every section.
-            // We're sorting from highest to lowest strain.
-            foreach (double strain in strains.OrderByDescending(d => d))
-            {
-                difficulty += strain * weight;
-                weight *= DecayWeight;
-            }
-
-            return difficulty * DifficultyMultiplier;
+            return strains.ExponentialWeightedSum(DifficultySumWeight) * DifficultyMultiplier;
```

Comment:
This should be ordering strains by descending (osu! tests are currently failing because this step is missing)

```suggestion
            return strains.SortedExponentialWeightedSum(DifficultySumWeight) * DifficultyMultiplier;
```

Reply:
Made the opposite change in a final visual pass & apparently I didn't rerun that test...